modules/apps/view-count/view-count-test/src/testIntegration/java/com/liferay/view/count/service/impl/test/ViewCountEntryLocalServiceTest.java

```diff
@@ -79,7 +79,8 @@ public void testLazyCreationWithRaceCondition() throws Throwable {
 
 		Assume.assumeFalse(
 			"HSQL does not allow concurrent Session assess, skip test.",
-			db.getDBType() == DBType.HYPERSONIC);
+			(db.getDBType() == DBType.HYPERSONIC) ||
```

Comment:
@yuhai You should put this in another assume statement, as the original assume is for HSQL.